test/fuzz/lua/test_engine.lua

```diff
@@ -788,6 +991,46 @@ local function random_key(space, idx)
     return key
 end
 
+-- Chooses random space fields and return their indexes,
+-- they are not ordered and may be repeated.
+-- NB: indexes are zero-based.
+local function random_space_field_ids(space)
+    local format_size = #space:format()
+    local fields = {}
+    local fields_num_max = format_size * 2
+    local fields_num = math.random(fields_num_max)
+    for _ = 1, fields_num do
+        table.insert(fields, math.random(0, format_size - 1))
+    end
+    return fields
+end
+
+-- Generates a batch as a Lua table in a format:
+-- {{field_name1, {<values>}}, {field_name2, {<values>}}, ...}.
+-- Fields are not repeated, nullable ones can be skipped.
+local function random_batch(space)
+    local format_size = #space:format()
+    local batch = {}
+    -- Randomly choose between small and big batch
+    local max_batch_size = oneof({10, 100})
```

Comment:
> Is it intentional?

Yep, here is the reason. We pass the generated batch to the INSERT_ARROW_OP and it is logged unconditionally from its size. Hence, too large batches will clutter up the logs. In `arrow_stream_op` function I log the batch manually so I can log only small batches.

Probably, we should automatically limit the length of log message so that I could generate huge batches here as well without cluttering logs.